TransferMethodRepository/Tests/TransferMethodRepositoryTests.swift

```diff
@@ -21,12 +21,18 @@ import HyperwalletSDK
 @testable import TransferMethodRepository
 import XCTest
 
-class RemoteTransferMethodRepositoryTests: XCTestCase {
+class TransferMethodRepositoryTests: XCTestCase {
+    private var factory: TransferMethodRepositoryFactory!
+    private var transferMethodRepository: TransferMethodRepository!
+
     override func setUp() {
         Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
+        factory = TransferMethodRepositoryFactory.shared
+        transferMethodRepository = factory.transferMethodRepository()
```

Comment:
You can directly use ``` TransferMethodRepositoryFactory.shared.transferMethodRepository() ``` instead of ``` factory.transferMethodRepository() ```

Reply:
Rename all test names to have full function name. Like instead of testCreate_bankAccount use testCreateTransferMethod_bankAccount